Source/Core/VideoCommon/VertexManagerBase.cpp

```diff
@@ -209,6 +209,7 @@ void VertexManager::Flush()
 				if (bpmem.tevind[i].IsActive() && bpmem.tevind[i].bt < bpmem.genMode.numindstages)
 					usedtextures[bpmem.tevindref.getTexMap(bpmem.tevind[i].bt)] = true;
 
+		TextureCache::UnbindTextures();
```

Comment:
This is basically zero additional overhead; I just needed to reorder the API calls so that I can bind textures in TextureCache::Load without worrying about messing up existing bindings.